onnx/defs/schema.h

```diff
@@ -566,7 +567,8 @@ class OpSchema final {
         "tensor(int64)",
         "tensor(float16)",
         "tensor(float)",
-        "tensor(double)"};
+        "tensor(double)",
+        "tensor(bfloat16)"};
```

Comment:
should we also add it in the "all_tensor_types" too?

Reply:
yes i added that too.